java/ql/lib/ext/org.apache.http.client.utils.model.yml

```
@@ -3,6 +3,7 @@ extensions:
       pack: codeql/java-all
       extensible: summaryModel
     data:
+      - ["org.apache.http.client.utils", "URIBuilder", True, "setPort", "(int)", "", "Argument[undefined]", "ReturnValue", "taint", "ai-manual"] # actually a value step
```

@kaeluka I think this indicates a bug in some of our tooling.
```suggestion
      - ["org.apache.http.client.utils", "URIBuilder", True, "setPort", "(int)", "", "Argument[this]", "ReturnValue", "taint", "ai-manual"] # actually a value step
```

---

yes, indeed. IIRC, I think I pinged you about this last time already

---

yes, indeed!